packages/fast-components-styles-msft/src/card/index.ts

```diff
@@ -0,0 +1,25 @@
+import designSystemDefaults, { IDesignSystem, withDesignSystemDefaults } from "../design-system";
+import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ICardClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { get } from "lodash-es";
+import Chroma from "chroma-js";
+import { elevation, ElevationMultiplier } from "../utilities/elevation";
+
+const styles: ComponentStyles<ICardClassNameContract, IDesignSystem> = (
+    config: IDesignSystem
+): ComponentStyleSheet<ICardClassNameContract, IDesignSystem> => {
+    const designSystem: IDesignSystem = withDesignSystemDefaults(config);
+    const foregroundColor: string = designSystem.foregroundColor;
+    const backgroundColor: string = designSystem.backgroundColor;
```

Comment:
nit: not sure it's necessary to declare these since they are only used once

Reply:
will update